src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.out;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.UUID;
+
+public class EnvelopeInfo {
+
+    @JsonProperty("container")
+    public final String container;
+
+    @JsonProperty("file_name")
+    public final String fileName;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonProperty("envelope_id")
+    public final String envelopeId;
```

Comment:
Let's make this attribute of type UUID. This we make code cleaner.

Reply:
done